README.md

```diff
@@ -408,6 +408,9 @@ Check the [constraint rule comparison matrix](rule_comparison.md) for language-s
   
   // x must conform to a well known regex for HTTP header values (via RFC 7230) 
   string x = 1 [(validate.rules).string.well_known_regex = HTTP_HEADER_VALUE];
+  
+// x must conform to a well known regex that guards against \r\n\0 (via Envoy's validation)
+  string x = 1 [(validate.rules).string.well_known_regex = ENVOY_VALID_HEADER];
```

Comment:
Could we maybe give this a name not tied to envoy? Something like: "Single line header", or "Strict Header"?